tests/torchtune/utils/test_precision.py

```diff
@@ -58,12 +58,18 @@ def test_error_bf16_unsupported(self, mock_verify):
 
     @pytest.mark.skipif(not cuda_available, reason="The test requires GPUs to run.")
     def test_set_float32_precision(self) -> None:
+        setattr(  # noqa: B010
+            torch.backends, "__allow_nonbracketed_mutation_flag", True
+        )
         _set_float32_precision("highest")
         assert torch.get_float32_matmul_precision() == "highest"
         assert not torch.backends.cudnn.allow_tf32
         assert not torch.backends.cuda.matmul.allow_tf32
 
         _set_float32_precision("high")
+        setattr(  # noqa: B010
+            torch.backends, "__allow_nonbracketed_mutation_flag", True
```

Comment:
is there a reason for adding "setattr" a second time and its placement here? Or doing it once at the start is enough?